src/NuGet.Core/NuGet.Commands/ClientCertificatesCommand/ModifyClientCertArgs.cs

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Security.Cryptography.X509Certificates;
+using NuGet.Common;
+
+namespace NuGet.Commands
+{
+    public abstract class ModifyClientCertArgs : BaseClientCertArgs
+    {
+        /// <summary>
+        ///     FindBy added to a storage client certificate source
```

Comment:
Extra spaces in xmldoc comments.

```suggestion
        /// FindBy added to a storage client certificate source
```